src/Backend/Core/Engine/Csv.php

```diff
@@ -54,6 +60,7 @@ public static function outputCSV(
         );
     }
 
+    // we are not using the line ending for now.
```

Comment:
Not sure if it's safe to drop this feature? Will all software handling csv's on multiple OS'es still be able to handle this?

Reply:
@WouterSioen Symfony's [CSV encoder](https://api.symfony.com/3.3/Symfony/Component/Serializer/Encoder/CsvEncoder.html) uses `fputcsv` to write CSV files so I assume it relies on PHP to properly detect the correct line endings. Not too sure if that's a good idea, though.

@Senjutsumo can you test on some different machines (OS X, Windows (with Excel), ...) to see if the CSV encoder always write readable files?

If it doesn't we could look at a CSV generating package on packagist.